.github/workflows/wrangler-action.yml

```diff
@@ -0,0 +1,42 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Deploy
+
+# Controls when the action will run.
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  deploy:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+      # install esBuild
+      - name: install esbuild
+        run : "npm install esbuild"
+      # build site
+      - name: build site
+        run: "npm run build"
+      # publish site
+      - name: build entry point
+        run : "node_modules/.bin/esbuild api/server.js --bundle --format=esm --outfile=api/index.js"
+      # Runs a single command using the runners shell
+      - name: Deploy to Cloudflare Workers with Wrangler
+        uses: cloudflare/wrangler-action@1.2.0
```

Comment:
In the end we do not need `wrangler`, I removed it from the project and deployed with node directly, you can check the code in the `dev/deploy.js` file.

We need to define a `CF_DEPLOY_TOKEN` thought, not sure how to bind secrets to env variables in github actions

Reply:
Ok